database/src/migrations/20230419151100_treatment_unit_updates.ts

```diff
@@ -32,19 +33,20 @@ export async function up(knex: Knex): Promise<void> {
       add check (reconnaissance_conducted in ('yes', 'no', 'not applicable'));
 
     -- Add new column
-    alter table treatment_unit 
```

Comment:
@cgarrettjones Was mistaken before, this new `implemented` column should have been on the `treatment` table, not the `treatment_unit` table. Everything else is the same.